include/bl32/payloads/tlk.h

```diff
@@ -36,18 +36,22 @@
 /*
  * Generate function IDs for the Trusted OS/Apps
  */
+#if !ERROR_DEPRECATED
 #define TLK_TOS_STD_FID(fid)	((fid) | 0x72000000 | (0 << 31))
 #define TLK_TA_STD_FID(fid)	((fid) | 0x70000000 | (0 << 31))
+#endif
```

Comment:
This file (and the Trusty) equivalent are public APIs. They (or a version of them) may be used by the OS, so this code aimed to maintain compatibility and provide a migration path, with a view to removing the old defines at a later date.

If it's considered unlikely that the OS will use these files directly (e.g. if there's no link between this copy and the OS's copy), then there may be no need to provide this migration path. Are you sure this is what you want?